core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/schedule/RJSTrainScheduleOptions.java

```diff
@@ -0,0 +1,14 @@
+package fr.sncf.osrd.railjson.schema.schedule;
+
+import com.squareup.moshi.Json;
+
+public class RJSTrainScheduleOptions {
+    /** Optional arguments for the standalone simulation */
+
+    @Json(name = "ignore_electrical_profiles")
+    public boolean ignoreElectricalProfiles;
```

Comment:
This isn't optional. I wouldn't trust the json parsers to pick the correct default value. 

I assume that this object will eventually have more values and they will all be optional.